searchorder/src/main/java/com/ibm/searchorder/OrderService.java

```diff
@@ -11,7 +11,7 @@
 @Service
 public class OrderService {
 	@Autowired
-	OrderRepository orderRepository;
+	OrderRepository orderRepository;	//DI
```

Comment:
good code

Reply:
Thank you :)

Reply:
can u check my change?

Reply:
You need to send me the review request for it, I am waiting for it.